webnode/src/components/storage/index.test.js

```diff
@@ -0,0 +1,14 @@
+import configureStore from 'redux-mock-store'
+import Storage, { onGenesisHashChange, onNumberOfChunksChange } from "components/storage/";
+
+const initialState = {}; 
+const mockStore = configureStore();
+let wrapper;
+let store;
+
+test('redux store', () => {
+  beforeEach(() => {
+	store = mockStore(initialState);
+	wrapper = shallow(<Storage store={store}/>);
+	})
```

Comment:
Does this file set things up for all the other tests?

Reply:
Its preparation before testing, we can use it for other tests